crates/context/config/src/client/env/mod.rs

```diff
@@ -0,0 +1,2 @@
+pub mod config;
+// pub mod proxy;
```

Comment:
I removed this to reduce the scope of the PR, `crates/context/config/src/client/env/proxy.rs` should be addressed independently